Documentation/ContentMapping/Index.rst

```diff
@@ -8,7 +8,7 @@ Content Mapping
 ===============
 
 Having a perfect visual appearance of a website is pointless, if the content
-editors enter in the backend is not shown at the frontend. In the last step, we
+entered into the backend is not visible in the frontend. In the last step, we
```

Comment:
In german you wouldn't enter content INTO the backend. You would enter IN the backend, meaning, while working IN the backend. So, seen through english glasses: What's correct?

Reply:
I'm unsure, shall we go with "entered in the backend"?